src/App.tsx

```diff
@@ -1,24 +1,253 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+
 import { UserWarning } from './UserWarning';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
+import { Notification } from './components/Notification';
+
+import { Todo } from './types/Todo';
+import { FilterBy } from './types/FilterBy';
+import { Errors } from './types/Errors';
 
-const USER_ID = 0;
+import { client } from './utils/fetchClient';
+import * as utils from './utils/utils';
+
+const USER_ID = 11246;
+const BASE_ADD_URL = '/todos';
+const URL_GET = `?userId=${USER_ID}`;
+const DELAY_ERROR = 3000;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [hasTodoLoaderIds, setHasTodoLoaderIds] = useState<number[]>([]);
```

Comment:
```suggestion
  const [todoLoaderIds, setTodoLoaderIds] = useState<number[]>([]);
```
use `has`, `is`, `can` prefixes for boolean values: [article](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)